src/InfluxDB2/Point.php

```diff
@@ -162,6 +162,20 @@ private function appendTags()
         foreach (array_keys($this->tags) as $key) {
             $value = $this->tags[$key];
 
+            if (!is_string($value) && null !== $value) {
+                if (
+                    is_scalar($value)  ||
+                    (is_object($value) && method_exists($value, '__toString()')) ||
+                    (\PHP_VERSION_ID >= 80000 && $value instanceof \Stringable)
+                ) {
+                    $value = (string) $value;
+                } else {
+                    throw new \InvalidArgumentException(
+                        sprintf('Tag value for key %s cannot be converted to string', $key)
+                    );
```

Comment:
Please, change the exception to a warning. This is a breaking change that could cause problems for other users.

Reply:
I would also suggest adding a unit test that verifies the exception is actually thrown. Because looking at the code, the parameter type hint should prevent the entire if-statement from ever executing.

Reply:
> Please, change the exception to a warning. This is a breaking change that could cause problems for other users.

done